src/hotspot/cpu/x86/c1_Defs_x86.hpp

```diff
@@ -44,7 +44,7 @@ enum {
   pd_nof_xmm_regs_frame_map = XMMRegister::number_of_registers,    // number of registers used during code emission
 
 #ifdef _LP64
-  #define UNALLOCATED 4    // rsp, rbp, r15, r10
+  #define UNALLOCATED 3    // rsp, r15, r10
```

Comment:
This affects pd_nof_caller_save_cpu_regs_frame_map below, but RBP is callee-saved, not caller-saved.

Reply:
Yes. I updated the patch.

I want to confirm, if we treat RBP as caller saved, is there any correctness problem?